src/api/services/adminServiceClient.ts

```diff
@@ -0,0 +1,57 @@
+import axios, { AxiosError } from 'axios';
+
+import { SSP_ADMIN_SERVICE_BASE_URL, SSP_ADMIN_SERVICE_CLIENT_KEY } from '../envars';
+import { getLoggers } from '../helpers/loggingHelpers';
+
+export type SharingListResponse = {
+  whitelist: number[];
+  whitelist_hash: number;
+};
+
+const adminServiceClient = axios.create({
+  baseURL: SSP_ADMIN_SERVICE_BASE_URL,
+  headers: {
+    common: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${SSP_ADMIN_SERVICE_CLIENT_KEY}`,
+    },
+  },
+});
+
+export const getSharingList = async (siteId: number): Promise<SharingListResponse> => {
+  try {
+    const response = await adminServiceClient.get(`/api/sharing/list/${siteId}`);
+    return response.data;
+  } catch (error: unknown) {
+    // if site don't have ACLs, currently it returns 404
+    if (error instanceof AxiosError && error.response?.status === 404) {
```

Comment:
Probably a better way would be to define 404 as a non-error status, so you only get an exception if something goes wrong.

The default axios `validateStatus` handler is:
```
validateStatus: (status) => {
  return status >= 200 && status < 300; // default
},
```

You could easily update that to also allow 404 as well as anything 2xx. Relevant docs: https://axios-http.com/docs/handling_errors